### PR TITLE
Fix 10814: Make OcModal vertically scrollable

### LINKED ISSUE #10814: Link creation modal not scrollable
## Describe the bug
If the browser height is small I'm not able to click on the button for creating submitting the link creation modal.

## Steps to reproduce
1. Have a small browser height
2. Create a file
3. Click on the "copy link" quick action in the file list
4. Type in a password so that the (rather lengthy) password rules appear

## Expected behavior
The controls of the modal (Cancel and Submit buttons) are always visible

## Actual behavior
The controls of the modal are not in the visible area anymore

## Screenshot
Screenshot 2024-04-22 at 22 41 59

## Additional context
Please fix on `stable-8.0`.

### PR DESCRIPTION
**Pull request by pascalwengerter**
## Related Issue
- Fixes #10814

### CODE REVIEW

**Review by JammingBen — CHANGES_REQUESTED** [review]
Modals now always show a scrollbar for me 😅 

image

Also please don't forget to target this to `stable-8.0`.

File: packages/design-system/src/components/OcModal/OcModal.vue
Comment (kulmann) [inline packages/design-system/src/components/OcModal/OcModal.vue:1]: wouldn't it make more sense to make only the modal body scrollable?

Comment (pascalwengerter) [reply packages/design-system/src/components/OcModal/OcModal.vue:1]: "It depends" 😉 assuming the title never exceeds one line I naively compute the "allowed" max height of the body. Or should I go for a more sophisticated solution?

**Review by JammingBen — COMMENTED** [review]
Still having some issues, the scrollbar is now showing for smaller modals as well:

image

**Review by kulmann — APPROVED** [review]
👍

**Review by JammingBen — APPROVED** [review]

**Pull request merged**